### ISSUE TITLE
Feature Request: keyboard shortcuts

### ISSUE THREAD

**Issue by thielepaul** [ItalyPaleAle/vscode-go-playground#1]
I love the idea of this extension! Would it be possible to add keyboard shortcuts for run and format?

**Comment by ItalyPaleAle** [comment]
Yes! That's a good idea. I need to figure out what shortcut I could assign that is unused however :(
